k8s/service_controller/model/service_broker.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+type ServiceBroker struct {
+	// json info from create docs
+	// http://apidocs.cloudfoundry.org/239/service_brokers/create_a_service_broker.html
+	Name         string `json:"name"`
+	BrokerURL    string `json:"broker_url"`
```

Comment:
I think Google is listing some things they would like to change, but my assumption is that for the immediate future we should align with what's in the CF docs/specs just to get things working - then we can see how people feel about changing things.